src/Global.asax.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Web;
+using System.Web.Routing;
+using System.Web.Http;
+
+namespace GroupDocs.Metadata.WebForms
+{
+    public class Global : HttpApplication
+    {
+        void Application_Start(object sender, EventArgs e)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private method 'Application_Start'.](https://app.codacy.com/gh/groupdocs-metadata/GroupDocs.Metadata-for-.NET-WebForms/pullRequest?prid=5102156)